tools/sphere.c

```diff
@@ -0,0 +1,71 @@
+// Mystical Sphere of Slooping
+// Gon
+// 2017-04
+
+inherit NewThingCode;
+inherit AnsiTellCode;
+
+#include <ansi.h>
+
+#define SLOOPEE "/usr/gon/tools/sloopee_shad"
```

Comment:
Call this SLOOPEE_SHAD and not just SLOOPEE. 

It isn't referring to the sloopee, right? That would be the player object. This is referring to the shadow object, so the #define should express that.